sound/soc/soc-pcm.c

```diff
@@ -1486,6 +1493,13 @@ static int dpcm_prune_paths(struct snd_soc_pcm_runtime *fe, int stream,
 		/* is there a valid CPU DAI widget for this BE */
 		widget = dai_get_widget(dpcm->be->cpu_dai, stream);
 
+		/*
+		 * The problem with VirtIO is, that as long as .playback_widget
+		 * or .capture_widget respectively is set, the back-end isn't
+		 * pruned. If we force the .{playback,capture}_widget = NULL,
+		 * then later that widget isn't included in the clean up.
+		 */
+
```

Comment:
@lyakh there's no code to justify the comment? Is this a TODO?

Reply:
@ranj063 the comment isn't well placed... No, it isn't a TODO, this has been fixed already, but the comment and the actual fix somehow drifted apart...